### PR TITLE
Feature/#29 text component

### PR DESCRIPTION
**Pull request by ginnaaceror**
Add text in all components with style components and change name of "assents file" for assets.

### CODE REVIEW

**Comment by ginnaaceror** [comment]
Hi @Israel-Laguan in the delivery component I think that the text in the header must be dynamic. This function is part of this issue or I put only the initial state?

**Comment by Israel-Laguan** [comment]
> Hi @Israel-Laguan in the delivery component I think that the text in the header must be dynamic. This function is part of this issue or I put only the initial state?

Not part of this ticket

**Comment by denisosuna** [comment]
looks good, what is left for this issue to be merged?

**Comment by Israel-Laguan** [comment]
I note this have a lot of small components. Why not just make one single Text component and pass some props like  color, size or similar?

**Comment by denisosuna** [comment]
> I note this have a lot of small components. Why not just make one single Text component and pass some props like color, size or similar?

Agree with this, @ginnaaceror to make that you could use functions in the style's definition and pass the props.
An example:
image

File: src/components/Text/Text.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  ${props => props.white && css`
+    ${[text.white, text.base]};
+  `}
+  ${props => props.small && css`
+    ${[text.white, text.xs]};
+  `}
+  
+  //Title modals
+  ${props => props.titleModal && css`
+  ${[text.primary, text.base]};
```
Comment (Israel-Laguan) [inline src/components/Text/Text.jsx]: I think the base is not needed here

File: src/components/Text/Text.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  `}
+
+  //Subtitles modals
+  ${props => props.danger && css`
+    ${[text.danger, text.xs]};
+  `}
+  ${props => props.primary && css`
+    ${[text.primary, text.xs, text['600']]};
```
Comment (Israel-Laguan) [inline src/components/Text/Text.jsx]: Please consider to change prop name as you are not only changing primary color

File: src/components/Text/Text.jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import styled, { css } from 'styled-components';
+import { text } from '../../styles/mixins';
+
+
+export const Text = styled.h1`
```
Comment (Israel-Laguan) [inline src/components/Text/Text.jsx]: can you change the tag to be `p` as is the most common case, please?

**Comment by Israel-Laguan** [comment]
When you finish to apply changes please quit Draft state to this PR

File: src/components/Text/Text.jsx [hunk 4]
```diff
@@ -2,7 +2,7 @@ import styled, { css } from 'styled-components';
 import { text } from '../../styles/mixins';
 
 
-export const Text = styled.h1`
+/* export const Text = styled.p`
```
Comment (Israel-Laguan) [inline src/components/Text/Text.jsx]: Please remove debug artifacts

**Review by Israel-Laguan — APPROVED** [review]

**Pull request merged**